packages/realm/package.json

```
@@ -65,7 +65,8 @@
     "build:node:prebuild:ia32": "wireit",
     "build:android": "wireit",
     "build:ios": "wireit",
-    "install": "prebuild-install --runtime napi || echo 'Failed to download prebuild for Realm'"
+    "install": "prebuild-install --runtime napi || echo 'Failed to download prebuild for Realm'",
+    "docs": "npx typedoc"
```

new command

---

Does this command depend on any of the others to be ran? In which case it could make sense to ["wireit up"](https://github.com/google/wireit/blob/main/README.md)?

---

when i was setting things up, it had to be run after the following commands in the root of the repo:

```
git submodule update --init --recursive
npm i
npm run bundle --workspace realm
```

can you point me in the right direction on what the wireit command should be based upon this?

---

Okay 👍 I can push a commit with the config?

---

Another nit, you don't actually need `npx` here, because NPM scripts are called with the recursive upward `node_modules/.bin` in `PATH`.

---

I didn't realise this was on a fork .. so I don't have rights to push.

Anyway, you just have to change the script to `wireit`

```suggestion
    "docs": "wireit"
```

and then include this in the list of `wireit` entries below the entry for `"build:ios"`:

```
    "docs": {
      "command": "typedoc",
      "dependencies": ["bundle"]
    }
```

---

Another nit, you didn't actually have to include `npx` here, because NPM scripts are called with the recursive upward `node_modules/.bin` in `PATH`.